sdk/containerregistry/azure-containers-containerregistry/TROUBLESHOOTING.md

```diff
@@ -0,0 +1,127 @@
+# Troubleshoot Azure Container Registry client library issues
+
+This troubleshooting guide contains instructions to diagnose frequently encountered issues while using the Azure Container Registry client library for Java.
+
+## General Troubleshooting
+
+Container registry service methods throw a [HttpResponseException][HttpResponseException] on failure.
+
+```java readme-sample-getProperties
+DefaultAzureCredential credential = new DefaultAzureCredentialBuilder().build();
+ContainerRepository containerRepository = new ContainerRegistryClientBuilder()
+    .endpoint(endpoint)
+    .credential(credential)
+    .buildClient()
+    .getRepository(repositoryName);
+try {
+    containerRepository.getProperties();
+} catch (HttpResponseException exception) {
+    // Do something with the exception.
+}
+```
+
+Azure Container Registry client library log errors by default. The errors and warnings in the logs or traces generally provide
+useful insights into what went wrong and sometimes include corrective actions to fix issues.
```

Comment:
What can we do to make `sometimes` into an always 😄

Reply:
In this specific case, it's unknown unknowns :)